tree/dataframe/test/dataframe_simple.cxx

```diff
@@ -726,12 +726,21 @@ TEST_P(RDFSimpleTests, StandardDeviationEmpty)
    auto stdDev = rd1.Define("b1", []() { return 0; }).StdDev("b1");
    EXPECT_DOUBLE_EQ(*stdDev, 0);
 }
-
+/*
 TEST(RDFSimpleTests, SumOfStrings)
 {
    auto df = RDataFrame(2).Define("str", []() -> std::string { return "bla"; });
    EXPECT_EQ(*df.Sum<std::string>("str"), "blabla");
 }
+*/
```

Comment:
this commit should come before the previous, otherwise the previous commit won't compile